files/zh-cn/web/javascript/reference/global_objects/array/includes/index.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Array.prototype.includes()
 slug: Web/JavaScript/Reference/Global_Objects/Array/includes
+page-type: javascript-instance-method
+browser-compat: javascript.builtins.Array.includes
```

Comment:
```suggestion
```
remove according to [guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)